nifi/nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/RouteOnAttribute.java

```diff
@@ -216,7 +233,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         }
 
         if (destinationRelationships.isEmpty()) {
-            logger.info(this + " routing " + flowFile + " to unmatched");
+            logger.info("{} routing {} to unmatched", new Object[]{ this, flowFile });
```

Comment:
We should just remove the 'this' all together because the ProcessorLogger will automatically prepend it for us anyway.